apps/src/code-studio/pd/application_dashboard/quick_view.jsx

```diff
@@ -85,13 +89,13 @@ export class QuickView extends React.Component {
     }
   }
 
-  load(regionalPartnerFilter = this.state.regionalPartnerFilter) {
+  load(regionalPartnerFilterValue = this.props.regionalPartnerFilter.value) {
```

Comment:
It appears `load` has this filter passed in both places it's called. Why do we need the default value?

Reply:
We don't!